packages/select/test/queryListTests.tsx

```diff
@@ -119,6 +129,59 @@ describe("<QueryList>", () => {
                 query: "123",
             });
             assert.deepEqual(filmQueryList.state().activeItem, TOP_100_FILMS[1]);
+
+            filmQueryList.unmount();
+        });
+    });
+
+    describe("activeItem state initialization", () => {
+        it("initializes to first filtered item when uncontrolled", () => {
+            const props: IQueryListProps<IFilm> = {
+                ...testProps,
+                // Filter down to only item at index 11, so item at index 11 should be
+                // chosen as default activeItem
+                itemPredicate: (_query, item) => item === TOP_100_FILMS[11],
+                query: "123",
+            };
+
+            const filmQueryList: ReactWrapper<IQueryListProps<IFilm>, IQueryListState<IFilm>> = mount(
+                <FilmQueryList {...props} />,
+            );
+
+            assert(filmQueryList.state().activeItem === TOP_100_FILMS[11]);
+
```

Comment:
nit: this whitespace is not necessary here since these are all the same logical block (setup & assert once). match the style of other tests.